### DISCUSSION TITLE
ungoogled chromium feature vs chromium

### DISCUSSION THREAD

**Discussion by vahidx4r4x** [Eloston/ungoogled-chromium#1576] [Q&A]
Hi
thanks for your great work

I have a question:

I'm looking forward to use the "Chromium" browser but I don't like being tracked by google,

So apart from "Disable functionality specific to Google domains" feature , the other two key features in the project are threats to my privacy?
I mean can google track me, store my data, IP Address, etc... if these features are available/On in browser?

**Comment by javathehunt** [comment]
I am about as technologically illiterate as you but to give you a response;

"I'm looking forward to use the "Chromium" browser..." 

- I assume you mean "Ungoogled-Chromium" " - because "Chromium" is a different browser. 

"the other two key features in the project are threats to my privacy?" 

- I don't understand what these other features are you refer to I must not be reading this in the context you mean it  to be. Nevertheless I have included some links below that briefly discuss the privacy features of Ungoogled.

Also when I started using Ungoogled I did a check after I disabled all extensions launched the browser and watched my firewall traffic for a period of time; if I remember correctly there was zero telemetry coming or going. Not true with Iridium and certainly not true with Brave. The only other browser that comes close is Tor but even that reached out to the tor project checking for updates.

Now on your Ungoogled; when you start turning on extensions you will see the comms start flowing with the outside world thru the firewall. Does that mean it's communicating with google? Certainly not but you need to understand your extensions. The one huge drawback on any chromium based browser is if you want to use a variety of extensions you are inevitably going to find yourself in the googe webstore and I am not aware of any addons in there being monitored for security like the firefox addon store. With that said - 1) If you do want to install apps from the webstore you should be downloading the crx files and installing offline; 2) The only real addons you need for enhanced privacy in my opinion are Ublock and Noscript and both of those can be downloaded from Github no need to visit the G webstore.

"I mean can google track me, store my data, IP Address, etc... if these features are available/On in browser?" 

- If you visit a google website or log into you gmail or allow cookies and trackers then yes google is simply shooting fish in a barrel; these are all things within your control so you do not need to be a victim. All the Ungoogled browser does is stop the browser from talking with google in the background and prevents the exchange of tracking data that is otherwise outside the users control. Ungoogled doesn't even check for updates - which is why you are on this website.


Read features et al;
https://github.com/Eloston/ungoogled-chromium


Section - It behaves in most regards just like Chromium but with the following main differences:
https://www.ghacks.net/2016/10/05/ungoogled-chromium-removes-google-traces-from-chromium/

**Reply by vahidx4r4x** [reply]
Hi
thanks for your help


* by "chromium" I mean the original "chromium browser project" not "un-googled chromium" or "google chrome"

* I assume that the key features in "un-googled chromium" are not present in original "chromium". So is there any threat to my privacy if I use "chromium"?
 [https://github.com/Eloston/ungoogled-chromium#feature-overview]

and my question is "Is there any chance of data collection from google if I use the original chromium browser?"

**Reply by javathehunt** [reply]
If Chromium was considered a "privacy" centric browser then people would not be using the less reliably updated Ungoogled-Chromium.

The answers to your questions are in the links I gave above.

**Reply by vahidx4r4x** [reply]
thanks i got my answers.

why this awesome project doesn't update frequently for windows ?
is there any other forks that is up to date?

**Reply by javathehunt** [reply]
I think its because the packages developers are volunteers therefor you get what you get when you get it. The Iridium browser for Debian platforms has not been updated in several years last I checked. It looks like Ungoogled for Debian is several updates behind and as well and appimage does not work on all platforms so I just dropped Ungoogled the other day and switched to Brave browser. Same with Bromite for the phone a bug since May still not fixed I switched the phone to Brave as well. Brave is not as anonymous as Ungoogled or Bromite but far better than chromium.

**Reply by thijmn0** [reply]
Brave is one of the most horrible browsers.
This website explains it nicely
https://digdeeper.neocities.org/articles/browsers
